event_registration_multi_qty/reports/report_event_registration.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ReportEventRegistration(models.Model):
+    """Events Analysis"""
+    _inherit = "report.event.registration"
+
+    def _sub_select(self):
```

Comment:
This won't work if https://github.com/odoo/odoo/pull/16633 is not merged, so we have to wait.